wiki/Ranking_Criteria/fr.md

```diff
@@ -297,3 +292,8 @@ Cette catégorie contient des déclarations d'allocation explicites de concepts
   - **Évitez d'avoir plusieurs sprites actifs alors qu'ils ne sont pas rendus.** Les sprites actifs continueront à traiter les commandes, qu'ils soient visibles ou non. Si c'est le cas pendant de longues périodes, instanciez de nouveaux sprites à la place, pour le moment où la visibilité est rétablie.
   - **Lorsque vous utilisez plusieurs commandes du même type sur un sprite, essayez de laisser au moins 16 ms entre leurs temps de démarrage.** 60 commandes par seconde sont souvent plus que suffisantes pour permettre à un sprite d'effectuer des transitions fluides dans une configuration moyenne. Ceci dans le but de réduire la taille des fichiers et les temps de chargement.
   - **Suppression des sprites activés par les déclencheurs après utilisation.** Les déclencheurs s'activent dès leur première commande possible et restent actifs jusqu'à la fin de la difficulté, c'est pourquoi il est préférable de les faire disparaître lorsqu'ils sont terminés.
+
+## Notes
+
+[^personnage-voix-acteur]: CV (Character Voice) est utilisé lorsque la musique est chantée dans le personnage par l'acteur de la voix du personnage. `VO` (Voice Over) est utilisé quand une musique est chantée dans le personnage par quelqu'un d'autre que l'acteur de la voix du personnage.
+[^dimensions-maximales]: Les arrière-plans qui sont également utilisés comme images de storyboard peuvent suivre [la règle des images de storyboard pour les dimensions maximales](/wiki/Ranking_Criteria#règles.7) à la place.
```

Comment:
also, the identifier here (e.g. `[^personnage-voix-acteur]`) should *not* be translated, we'll prob add that to ASC soon